lib/puppet/file_serving/base.rb

```diff
@@ -80,6 +80,9 @@ def to_data_hash
   end
 
   def self.absolute?(path)
-    Puppet::Util.absolute_path?(path, :posix) or (Puppet.features.microsoft_windows? and Puppet::Util.absolute_path?(path, :windows))
+    # TODO: this is the one spot that will be tricky
```

Comment:
Need to find an answer for this, so that we don't need to perform multiple style `absolute_path?` checks that are OS agnostic.